From e8a04c1d84d2ebd0dbdf7bda26d7a22017100586 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Thu, 9 Jul 2020 00:24:57 -0400
Subject: [PATCH 50/62] Remove a couple of incorrect license claims.

A certain someone's default editor template leaked in to a couple of
source files, and claims they're GPL licensed.  They're not.

Signed-off-by: Peter Jones <pjones@redhat.com>
Upstream-commit-id: 476cbff1110
---
 errlog.c | 3 ---
 mok.c    | 2 --
 2 files changed, 5 deletions(-)

diff --git a/errlog.c b/errlog.c
index eebb266d396..6669c800233 100644
--- a/errlog.c
+++ b/errlog.c
@@ -1,10 +1,7 @@
 /*
  * errlog.c
  * Copyright 2017 Peter Jones <pjones@redhat.com>
- *
- * Distributed under terms of the GPLv3 license.
  */
-
 #include "shim.h"
 
 static CHAR16 **errs = NULL;
diff --git a/mok.c b/mok.c
index 59630e74425..089ea6bfc9a 100644
--- a/mok.c
+++ b/mok.c
@@ -1,8 +1,6 @@
 /*
  * mok.c
  * Copyright 2017 Peter Jones <pjones@redhat.com>
- *
- * Distributed under terms of the GPLv3 license.
  */
 
 #include "shim.h"
-- 
2.26.2

